modular_skyrat/modules/new_meteors/code/new_meteors.dm

```diff
@@ -0,0 +1,53 @@
+#define METEOR_WAVE_MIN_NOTICE 210
+#define METEOR_WAVE_MAX_NOTICE 240
+#define METEOR_WAVE_DURATION 37
```

Comment:
Might want comments saying how long these are in real time rather than code time (I do not know bc I am stupid)

Reply:
Sure